demisto_sdk/commands/generate_yml_from_python/README.md

```diff
@@ -0,0 +1,1510 @@
+# Generate YML from python
```

Comment:
we're currently supporting only integrations right? if so, lets mention it please